p2p/host/hostv2/hostv2.go

```diff
@@ -112,16 +112,15 @@ func (host *HostV2) GroupReceiver(group p2p.GroupID) (
 
 // AddPeer add p2p.Peer into Peerstore
 func (host *HostV2) AddPeer(p *p2p.Peer) error {
+	if p.PeerID == "" {
+		host.logger.Error("AddPeer PeerID is EMPTY", "addrs", p.Addrs)
+		return fmt.Errorf("AddPeer error: peerID is empty")
```

Comment:
this is redundant.  If you return fmt.errorf, no need to add a log line here.

Reply:
i'll remove that - it was someone else code and i just moved it here